tests/validations.sh

```diff
@@ -225,11 +225,15 @@ cd "$TEST_WORKSPACE/$mono_repo_dir"
 
 # helm_repo_url="$AZDO_ORG_URL/$AZDO_PROJECT/_git/$helm_charts_dir"
 local_repo_url="$AZDO_ORG_URL/$AZDO_PROJECT/_git/$mono_repo_dir"
-spk service create $FrontEnd $FrontEnd -d $services_dir -p "chart" -g $local_repo_url -b master >> $TEST_WORKSPACE/log.txt
```

Comment:
@yradsmikham Perhaps we should have a separate instance in the test for this. If I'm understanding we are now not going to use the main code path for `spk service create`

Reply:
I feel like this should be fine since it's not really deviating from the "main code path". It just adds another step to it. 

But also, adding another instance of it would make the tests longer because we would essentially need to tear it down, just to build it back up in order to test a small component of it.

@mtarng what are your thoughts on this?

Reply:
Right. Currently this test is just adding a build parameter that is being read by `acr build` but ignored by our sample Dockerfile anyways. The sanity check is to make sure they're being added into our build script and being passed to acr.

Another alternative would require us to create a new Dockerfile for another service that we'd have to pull down after building and validate the build args in the container. I think for now, what we have should cover the need. Though I do agree we should followup with a task to test a container with the build arguments.